tests/conftest.py

```diff
@@ -141,6 +143,19 @@ def pytest_configure():
 init_args = {}
 
 
+@pytest.fixture(scope="function")
+def logged_in_account():
```

Comment:
When we're running locally, are we expected to set `RH_TOKEN` and `RH_USERNAME`?